source/_components/tradfri.markdown

```diff
@@ -20,6 +20,7 @@ For this to work, you need to install a modified lib-coap library:
 
 ```bash
 $ sudo apt-get install libtool
+$ sudo apt-get install autoconf
```

Comment:
Wouldn't it be an idea to switch to `sudo apt-get install build-essential` as I'm not sure if Debian/Ubuntu/armbian are shipping the compiler?